src/khoj/routers/api_chat.py

```diff
@@ -707,6 +745,8 @@ def collect_telemetry():
         ## Extract Document References
         compiled_references: List[Any] = []
         inferred_queries: List[Any] = []
+        file_filters = conversation.file_filters if conversation and conversation.file_filters else []
```

Comment:
Where are these `file_filters` used?

Reply:
They're used by the summarization and notes lookup portions. They essentially limit what parts of the "brain"/knowledge base the conversation has access to.